autofit/non_linear/paths/directory.py

```diff
@@ -156,7 +156,7 @@ def save_all(self, search_config_dict, info, pickle_files):
     @make_path
     def _pickle_path(self) -> str:
         """
-        This is private for a reason - use the save_object etc. methods to save and load pickles
+        This is private for a reason, use the save_object etc. methods to save and load pickles

```

Comment:
You can make it _really_ private with a double underscore

Reply:
You wrote that code and comment lol